--- compare/good/pages/index.js	2020-11-12 09:07:13.331719489 +0100
+++ compare/bad/pages/index.js	2020-11-12 09:07:44.472131217 +0100
@@ -11,6 +11,53 @@
 // EXPORTS
 __webpack_require__.d(__webpack_exports__, "default", function() { return /* binding */ Index; });
 
+// CONCATENATED MODULE: ./node_modules/@babel/runtime/helpers/esm/jsx.js
+var REACT_ELEMENT_TYPE;
+function _createRawReactElement(type, props, key, children) {
+  if (!REACT_ELEMENT_TYPE) {
+    REACT_ELEMENT_TYPE = typeof Symbol === "function" && Symbol["for"] && Symbol["for"]("react.element") || 0xeac7;
+  }
+
+  var defaultProps = type && type.defaultProps;
+  var childrenLength = arguments.length - 3;
+
+  if (!props && childrenLength !== 0) {
+    props = {
+      children: void 0
+    };
+  }
+
+  if (childrenLength === 1) {
+    props.children = children;
+  } else if (childrenLength > 1) {
+    var childArray = new Array(childrenLength);
+
+    for (var i = 0; i < childrenLength; i++) {
+      childArray[i] = arguments[i + 3];
+    }
+
+    props.children = childArray;
+  }
+
+  if (props && defaultProps) {
+    for (var propName in defaultProps) {
+      if (props[propName] === void 0) {
+        props[propName] = defaultProps[propName];
+      }
+    }
+  } else if (!props) {
+    props = defaultProps || {};
+  }
+
+  return {
+    $$typeof: REACT_ELEMENT_TYPE,
+    type: type,
+    key: key === undefined ? null : '' + key,
+    ref: null,
+    props: props,
+    _owner: null
+  };
+}
 // EXTERNAL MODULE: ./node_modules/react/index.js
 var react = __webpack_require__("q1tI");
 var react_default = /*#__PURE__*/__webpack_require__.n(react);
@@ -272,15 +319,16 @@
 
 
 
+
 var __jsx = react_default.a.createElement;
 var style = {
   color: '#6b61ff'
 };
-var H1 = function H1() {
+var components_H1 = function H1() {
   console.log('>>>> H1 <<<<<');
-  return __jsx("h1", {
+  return /*#__PURE__*/_createRawReactElement("h1", {
     style: style
-  }, "Hello!");
+  }, void 0, "Hello!");
 };
 function LinkExternal(_ref) {
   var href = _ref.href,
@@ -297,6 +345,7 @@
 // CONCATENATED MODULE: ./src/content.md
 
 
+
 var content_jsx = react_default.a.createElement;
 
 
@@ -313,7 +362,7 @@
   return createElement(MDXLayout, extends_extends({}, layoutProps, props, {
     components: components,
     mdxType: "MDXLayout"
-  }), createElement("h1", null, "Header 1"), createElement("p", null, createElement("a", extends_extends({
+  }), /*#__PURE__*/_createRawReactElement("h1", {}, void 0, "Header 1"), /*#__PURE__*/_createRawReactElement("p", {}, void 0, createElement("a", extends_extends({
     parentName: "p"
   }, {
     "href": "http://example.com"
@@ -322,19 +371,20 @@
 ;
 MDXContent.isMDXComponent = true;
 // CONCATENATED MODULE: ./src/pages/index.jsx
+
 var pages_jsx = react_default.a.createElement;
 
 
 
 
 var pages_components = {
-  h1: H1,
+  h1: components_H1,
   a: LinkExternal
 };
 function Index() {
-  return pages_jsx(esm_MDXProvider, {
+  return /*#__PURE__*/_createRawReactElement(esm_MDXProvider, {
     components: pages_components
-  }, pages_jsx(MDXContent, null));
+  }, void 0, /*#__PURE__*/_createRawReactElement(MDXContent, {}));
 }
 
 /***/ }),
